iotlabsshcli/open_a8.py

```diff
@@ -59,143 +57,115 @@ def _nodes_grouped(nodes):
 _MAKE_EXECUTABLE_CMD = 'chmod +x {}'
 _RUN_SCRIPT_CMD = 'screen -S {screen} -dm bash -c \"{path}\"'
 _QUIT_SCRIPT_CMD = 'screen -X -S {screen} quit'
+_REMOTE_A8_DIR = 'A8/.iotlabsshcli'
 
 
 def flash_m3(config_ssh, nodes, firmware, verbose=False):
-    """Flash the firmware of M3 of open A8 nodes."""
-    # Configure ssh and remote firmware names.
+    """ Flash the firmware of M3 co-microcontroller """
```

Comment:
You are changing all docstring but please keep as they were before (no surrounding space and a period). Have a look at [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).